storagegroup/storagegroup.go

```diff
@@ -206,3 +211,77 @@ func formatCheck(v2 *storagegroup.StorageGroup) error {
 
 	return nil
 }
+
+// ReadFromObject assemble StorageGroup from a regular
+// Object structure. Object must contain unambiguous information
+// about its expiration epoch, otherwise behaviour is undefined.
+//
+// Returns any error appeared during storage group parsing.
+func ReadFromObject(sg *StorageGroup, o objectSDK.Object) error {
+	err := sg.Unmarshal(o.Payload())
+	if err != nil {
+		return fmt.Errorf("could not unmarshal object: %w", err)
+	}
+
+	var expObj uint64
+
+	for _, attr := range o.Attributes() {
+		if attr.Key() == objectV2.SysAttributeExpEpoch {
+			expObj, err = strconv.ParseUint(attr.Value(), 10, 64)
+			if err != nil {
+				return fmt.Errorf("could not get expiration from object: %w", err)
+			}
+
+			break
+		}
+	}
+
+	// Supporting deprecated functionality.
+	// See https://github.com/nspcc-dev/neofs-api/pull/205.
+	if expSG := sg.ExpirationEpoch(); expObj != expSG {
+		return fmt.Errorf(
+			"expiration does not match: from object: %d, from payload: %d",
+			expObj, expSG)
+	}
+
+	return nil
+}
+
+// WriteToObject writes StorageGroup to a regular
+// Object structure. Object must not contain ambiguous
+// information about its expiration epoch.
+func WriteToObject(sg StorageGroup, o *objectSDK.Object) {
```

Comment:
Minor wish: if we learn from the experience of `copy`, `io` package and some other places, the entry - `Object` - element is the first argument.